terraform/modules/galileo-eks/examples/variables.tf

```diff
@@ -13,3 +13,33 @@ variable "region" {
   type        = string
   default     = "us-west-2"
 }
+
+variable "create_ml_node_group" {
+  description = "Set to true to launch ML node group / workers instances"
+  type        = bool
+  default     = false
+}
+
+variable "ml_node_size" {
+  description = "ML/GPU node size. Defaults to `g4dn.2xlarge`"
+  type        = string
+  default     = "g4dn.2xlarge"
+}
+
+variable "create_rds_postgres_cluster" {
+  description = "Specifies whether create RDS postgres cluster. Defaults to `true`"
+  type        = bool
+  default     = true
```

Comment:
Can we default this to false, in case some customers want self-hosted db? We can always ask them to enable this explicitly if needed

Reply:
I will make it false

Reply:
I think defaulting to RDS for AWS clusters is better.

Reply:
The concern is that if a customer doesn't need RDS, they may miss this variable and forget to disable it. But conversely they can notice this and enable it explicitly

Reply:
What I'm suggesting is: The default _should_ be that they use RDS with S3. And if they don't want it, they would need to explicitly disable it.

Anyway, I'm fine with us adding it this way for now but would encourage reconsidering that. RDS with S3 is just a nicer pathway for AWS customers.